methods.rb

```diff
@@ -0,0 +1,96 @@
+module Enumerable
+  def my_each
```

Comment:
Additional Requirements for this method:
it should:
- return an Enumerator if no block is given